services/download/src/main/java/app/coronawarn/server/services/download/FederationBatchProcessor.java

```diff
@@ -44,10 +44,10 @@
   private final FederationGatewayDownloadService federationGatewayDownloadService;
   private final DownloadServiceConfig config;
   private final ValidFederationKeyFilter validFederationKeyFilter;
-  
+
   // This is a potential memory-leak if there are very many batches
-  // This is an intentional decision: 
-  // We'd rather run into a memory-leak if there are too many batches 
+  // This is an intentional decision:
+  // We'd rather run into a memory-leak if there are too many batches
   // than run into an endless loop if a batch-tag repeats
```

Comment:
please turn this into real JavaDoc comment

Reply:
I've made this change